Resources/Views/pages/location.leaf

```diff
@@ -0,0 +1,102 @@
+#extend("layout/base")
+
+#export("content") {
+
+    <div class="container">
+        <div class="first-content">
+
+            <div class="more-info">
+                <div class="row">
+                    <div class="col-md-12">
+                        <h1 class="text-center fc-orange">Location</h1>
+                        <h3 class="text-center fc-white">
+                            In one of the most successful high-technology locations in Germany
+                        </h3>
+                        <div class="container text-left m-t-30">
+                            <div class="row">
+                                <div class="col-xs-12 col-sm-6">
+                                    <h3> Location </h3>
+                                    <h4>
+                                        The conference will take place at <a href="https://conventions.wista.de/en/rooms-locations/" target="_blank"> WISTA conventions </a> in the heart of the Technology Park of Adlershof, Berlin. Close to Schönefeld airport and with easy access to restaurants and accommodations, we found this venue to be perfect for this conference.
+                                    </h4>
+                                </div>
+                                <div class="col-xs-12 col-sm-6">
+                                    <h3> &nbsp; </h3>
+                                    <h4>
+                                        <script src="https://maps.googleapis.com/maps/api/js?v=3.exp"></script>
+                                        <div style='overflow:hidden;height:220px;width:500px;'>
+                                            <div id='gmap_location' style='height:220px;width:500px;'></div>
+                                            <style>#()gmap_location img{max-width:none!important;background:none!important}</style>
+                                        </div>
+                                        <script type='text/javascript'>function init_map(){var myOptions = {zoom:16,center:new google.maps.LatLng(52.432524280286664,13.538247942358339),mapTypeId: google.maps.MapTypeId.ROADMAP};map = new google.maps.Map(document.getElementById('gmap_location'), myOptions);marker = new google.maps.Marker({map: map,position: new google.maps.LatLng(52.432524280286664,13.538247942358339)});infowindow = new google.maps.InfoWindow({content:'<strong>WISTA Conventions</strong><br> Rudower Chaussee 17<br>'});google.maps.event.addListener(marker, 'click', function(){infowindow.open(map,marker);});infowindow.open(map,marker);}google.maps.event.addDomListener(window, 'load', init_map);
```

Comment:
Could use Apple maps 😉 


But leave it as Google!

Reply:
Haha yeah haven't had a look into it but basically why not (sneaking it in, in a future update) 😄